test/primitives.jl

```diff
@@ -174,22 +174,56 @@ module TestPrimitives
 
 # ----- test dropnull --------------------------------------------------------#
 
-    # dropnull(X::NullableArray)
+    # dropnull(X::NullableVector)
     z = NullableArray([1, 2, 3, 'a', 5, 'b', 7, 'c'], Int, Char)
     @test dropnull(z) == [1, 2, 3, 5, 7]
 
-    # dropnull(X::AbstractArray)
+    # dropnull(X::AbstractVector)
     A = Any[Nullable(1), Nullable(2), Nullable(3), Nullable(), Nullable(5),
-            Nullable(), Nullable(7), Nullable()]
+           Nullable(), Nullable(7), Nullable()]
```

Comment:
Is the proper indent like this where the continuation lines up with the `[` (as seen later in this same document [here](https://github.com/JuliaStats/NullableArrays.jl/blob/master/test/primitives.jl#L242) and [here](https://github.com/JuliaStats/NullableArrays.jl/blob/master/test/primitives.jl#L288)), or is the goal to have the elements line up (`Nullable` lines up with `Nullable`)?